docs/source/daml-script/index.rst

```diff
@@ -199,6 +199,30 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
 
 If you open Navigator, you can now see the contracts that have been created.
 
+Migrating from Scenarios to DAML Script for Ledger Initialization
+=================================================================
+
+Existing scenarios that you used for ledger initialization can be
+translated to DAML script but there are a few things to keep in mind:
```

Comment:
Should there be a separate section introducing `init-script` as a concept before going into migration, or is this planned for later when DAML script is stabilized?

Reply:
Good point! I’ve added a section on that. PTAL

Reply:
Thank you, that looks great!